nservicebus/bridge/configuration_failed-messages_bridge_[4,).partial.md

```diff
@@ -0,0 +1,8 @@
+### Error queue
+
+By default, when the bridge transfers a message to the ServiceControl error queue it will attempt to translate the [`NServiceBus.ReplyToAddress`](/nservicebus/messaging/headers.md#messaging-interaction-headers-nservicebus-replytoaddress) message header.
+It can only do this successfully if the endpoint in the ReplyToAddress header is registered with the bridge, therefore all endpoints need to be registered with the bridge otherwise translation of the `NServiceBus.ReplyToAddress` will fail, and the message will be moved to the `bridge.error` queue instead of correctly transferred to the ServiceControl `error` queue.
```

Comment:
```suggestion
It can only do this successfully if the address in the `ReplyToAddress` header maps to some endpoint registered with the bridge. Therefore all endpoints in the system need to be registered with the bridge. Otherwise, the translation of the `ReplyToAddress` header might fail for some messages, which in turn will be moved to [the bridge error queue](/nservicebus/bridge/configuration.md#recoverability-error-queue).
```